SpywareFilter/sections/specific.txt

```diff
@@ -23,6 +23,18 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+||w.atwiki.jp/*/google_analytics.js
+||w.atwiki.jp/*/counter.js
+||sentry.afreecatv.com/api/5/store/?sentry_key=
```

Comment:
sentry is not a tracker. Used by developers for bug tracking.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/cd7fb346d60350249e908a5f103df5d8f5b7d17d